ui/app/serializers/deployment.js

```diff
@@ -8,18 +8,25 @@ export default ApplicationSerializer.extend({
   },
 
   normalize(typeHash, hash) {
```

Comment:
Any reason the first argument is called `typeHash` and not `typeClass` [as in the JSONSerializer](https://www.emberjs.com/api/ember-data/3.3/classes/DS.JSONSerializer/methods/normalize?anchor=normalize)?

Reply:
Nope. At some point very early on I used `typeHash` and I have just rolled with it. It bothers me too, but apparently not quite enough to update all instances of it.